Modules/Filtering/Smoothing/test/itkDiscreteGaussianImageFilterTest.cxx

```diff
@@ -72,7 +72,19 @@ itkDiscreteGaussianImageFilterTest(int, char *[])
     filter->SetMaximumError(.01);
     filter->SetMaximumKernelWidth(32);
     filter->SetFilterDimensionality(Dimension);
-    filter->SetUseImageSpacing(true);
+
+    bool useImageSpacing = true;
```

Comment:
Same as above.

Reply:
In this case, contrary to [here](https://github.com/InsightSoftwareConsortium/ITK/pull/2286#discussion_r578092375), I grant that the previous commit did not add an input argument to the test, and I now realize that some compilers might flag the blocks as never being reached, which is annoying. 

So I added in 8e4315a a test with each boolean value taking advantage of the fact that there is no comparison to a baseline in this case. That can be added at a latter time and see if the results make sense.